package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "jupyterlab_sparkmonitor",
-  "version": "1.1.0",
+  "name": "jupyterlab_sparkmonitor-s",
+  "version": "1.2.0",
```

Comment:
Whoops, I think this change accidentally slipped in. Could we change the name and version back to their original values? The name probably was an accidental change and I'll update the version in master when I publish :)

Reply:
done